test/index.test.js

```diff
@@ -48,6 +73,12 @@ describe('Error', () => {
     return validateOptions('test/fixtures/schema.json', options, '{Name}');
   };
 
+  test('should throw non objects', () => {
```

Comment:
@evilebottnawi @sokra I'd like to verify that this is not a breaking change.

Reply:
@hulkish before it was throw other value?

Reply:
same test case for prior code yields:
```
 FAIL  test/index.test.js
  Valid
    ✓ should pass, uncached (7ms)
    ✕ should pass, cached
  Error
    ✓ should throw non objects (1ms)
    ✓ Throws (1ms)
    ✓ Errors (3ms)
    Messages
      ✓ Default
      ✓ Customized (4ms)

  ● Valid › should pass, cached

    SyntaxError: Unexpected token u in JSON at position 0
        at JSON.parse (<anonymous>)

      at validateOptions (src/validateOptions.js:28:19)
      at validate (test/index.test.js:31:12)
      at Object.test (test/index.test.js:51:12)
          at new Promise (<anonymous>)
          at <anonymous>
```